XRTK-Core/Packages/com.xrtk.core/package.json

```diff
@@ -10,7 +10,7 @@
     "Mixed Reality",
     "DI"
   ],
-  "version": "0.2.11",
+  "version": "0.2.12",
```

Comment:
Let's revert the version until our next release

Reply:
It's a breaking change and WMR won't be able to build without making a releas of core.

Reply:
Ah right, well that means we will need to do another release, but we just did one last night 😅

Reply:
I am fine with holding of merging for a while and keeping the PR open. I am planning some more anyways.